packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -267,7 +267,7 @@
 		.textfield-input {
 			background: transparent;
 			box-shadow: none;
-			padding: 1rem 1.75rem 0 1rem;
+			padding: 1rem 2.5rem 0 1rem;
```

Comment:
Why do you need to change the right padding ?

Reply:
Currently the spacing overlaps the cross and as a result we have the ellipsis that comes to rest on the cross